cmd/scollector/collectors/program.go

```diff
@@ -151,12 +178,20 @@ Loop:
 			Value:     val,
 			Tags:      opentsdb.TagSet{"host": util.Hostname},
 		}
-		for _, tag := range sp[3:] {
-			tags, err := opentsdb.ParseTags(tag)
+		for _, field := range sp[3:] {
+			if strings.HasPrefix(field, "meta.") {
```

Comment:
I thought I commented on this already, sorry. My idea was to have a `|` after everything that exists already, and then you can do `Metakey1="value"  Metakey2="value"` on the same line. I thought about adding another line, but I thought that might be problematic because the order of lines being flushed needs to be ensured, not awesome if you have something with multiple threads.